legal-api/src/legal_api/models/business.py

```diff
@@ -50,7 +49,7 @@
 from .user import User  # noqa: F401,I003 pylint: disable=unused-import; needed by the SQLAlchemy backref
 
 
-class Business(db.Model):  # pylint: disable=too-many-instance-attributes,disable=too-many-public-methods
+class Business(db.Model, Versioned):  # pylint: disable=too-many-instance-attributes,disable=too-many-public-methods
```

Comment:
to use new versioning, adding `Versioned` class is probably the main thing that needs to be added for a given model